### PR TITLE
update preimage-related docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the preimage-related docs for the RT2000 breaking changes. Reference: [Moonbeam PR description](https://github.com/PureStake/moonbeam/pull/1933) and [Substrate PR description](https://github.com/paritytech/substrate/pull/11649)

👉 Democracy Pallet Changes:
- note_preimage is moved to pallet-preimages
- note_preimage_imminent has been removed and there is no equivalent
- propose - now accepts a Bounded<Call> argument instead of a proposal hash. This can be defined by a proposal hash + length of the proposal, or the call to be executed per se
- Preimages storage item now lives in pallet-preimages

👉A new Preimage Pallet page has been added.

👉 Democracy Precompile Changes:
- The two functions that were not previously supported are now supported so I made appropriate changes there
- Added the associated Events
- Updated the preimage instructions as preimages need to be created now before a proposal can be submitted and it must be done from the new preimages page (Refer to [PR description](https://github.com/PureStake/moonbeam/pull/1933))

👉 How to Submit a Proposal Changes:
- Add all new images
- Submit preimages from the new preimages page

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [x] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [x] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

https://github.com/PureStake/moonbeam-docs-cn/pull/211

### After Translation Requirements

- [X] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Remove all of the governance proposals images under the following directory:
- `images/tokens/governance/proposals/for-cn/`

### CODE REVIEW

File: builders/pallets-precompiles/precompiles/democracy.md [hunk 1]
```diff
@@ -119,7 +125,16 @@ On the next screen, take the following steps:
 ![Get the encoded proposal](/images/builders/pallets-precompiles/precompiles/democracy/democracy-5.png)
 
 !!! note
-     You should NOT sign and submit the transaction here. You will submit this information via the **notePreimage** function in a later step.  
+     You should NOT sign and submit the transaction here. You will submit this information via the **notePreimage** function in the next step.  
+
+Now you can take the encoded proposal that you got from [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmoonbeam-alpha.api.onfinality.io%2Fpublic-ws#/democracy){target=_blank} and submit it via the `notePreimage` function of the democracy precompile. Despite its name, the preimage is not required to be submitted before the proposal. However, submitting the preimage is required before a proposal can be enacted. To submit the preimage via the `notePreimage` function, take the following steps:
```
Comment (albertov19) [inline builders/pallets-precompiles/precompiles/democracy.md:130]: In the near future, I would like us to move to examples with Polkadot.js SDK when it makes sense. I think this is an end user oriented tutorial more than anything else, so it maybe makes sense to keep it as such, but not sure what you think

Comment (albertov19) [inline builders/pallets-precompiles/precompiles/democracy.md:130]: I feel in the section where you say:

```
First, you can get the preimage hash by navigating to the Governance tab of [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmoonbeam-alpha.api.onfinality.io%2Fpublic-ws#/democracy) and selecting Preimages from the dropdown. From the Preimages page, click on + Add preimage, and take the following steps:
```

We are either missing an image taking you through the process, or you can put an Polkadot.js Apps URL direclty

**Review by albertov19 — CHANGES_REQUESTED** [review]
Looks good! Left some feedback :)

File: builders/pallets-precompiles/pallets/preimage.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+This guide will provide an overview of the extrinsics, storage methods, and getters for the pallet constants available in the preimage pallet on Moonbeam.
+
+## Preimage Pallet Interface {: #preimage-pallet-interface }
+
+### Extrinsics {: #extrinsics }
+
+The preimage pallet provides the following extrinsics (functions):
+
+- **notePreimage**(encodedProposal) - registers a preimage for an upcoming proposal given the encoded preimage of a proposal. If the preimage was previously requested, no fees or deposits are taken for providing the preimage. Otherwise, a deposit is taken proportional to the size of the preimage. Emits a `Noted` event
+- **requestPreimage**(bytes) - requests a preimage to be uploaded to the chain without paying any fees or deposits. If the preimage requests has already been provided on-chain by a user, their the related deposit is unreserved, and they no longer control the preimage. Emits a `Requested` event
```
Comment (eshaben) [inline builders/pallets-precompiles/pallets/preimage.md:26]:
```suggestion
- **requestPreimage**(bytes) - requests a preimage to be uploaded to the chain without paying any fees or deposits. If the preimage request has already been provided on-chain by a user, their related deposit is unreserved, and they no longer control the preimage. Emits a `Requested` event
```

File: builders/pallets-precompiles/precompiles/democracy.md [hunk 3]
```diff
@@ -119,7 +125,16 @@ On the next screen, take the following steps:
 ![Get the encoded proposal](/images/builders/pallets-precompiles/precompiles/democracy/democracy-5.png)
 
 !!! note
-     You should NOT sign and submit the transaction here. You will submit this information via the **notePreimage** function in a later step.  
+     You should NOT sign and submit the transaction here. You will submit this information via the **notePreimage** function in the next step.  
+
```
[the remark this answers is not in the document]
Comment (eshaben) [reply builders/pallets-precompiles/precompiles/democracy.md:129]: Added a note to the preimage pallet page

**Comment by eshaben** [comment]
Applied all of the feedback and switched around the images so that Xinxin only has to worry about removing images that are no longer needed once translations are done

Had a question about one of the democracy precompile methods, so asked J0 and waiting on an answer before I'll request a review again

**Comment by eshaben** [comment]
Ok this is good to be reviewed again!

**Review by albertov19 — APPROVED** [review]
Great! Ship it 🚢

**Pull request merged**